test/FunctionalTestUtils/TelemetryTestsBase.cs

```diff
@@ -45,7 +45,7 @@ public void ValidateBasicRequest(InProcessServer server, string requestPath, Req
             Assert.Equal(expected.Name, actual.Name);
             Assert.Equal(expected.Success, actual.Success);
             Assert.Equal(expected.Url, actual.Url);
-            Assert.InRange(actual.Timestamp, testStart, DateTimeOffset.Now);
+            InRange(actual.Timestamp, testStart, DateTimeOffset.Now);
```

Comment:
if you changed to high precision clock for testStart, why not use the same high precision clock for endClock as well ?